README.md

```diff
@@ -88,6 +89,16 @@ Refer to the gradle doc for [PmdExtension](https://docs.gradle.org/current/dsl/o
 
 The default `pmdRuleSetFile`, `findBugsExcludeFilterFile`, and `lintConfig` files can be found [here](src/main/resources).
 
+### Publishing
+This project is set to publish to the [Gradle plugins repository](https://plugins.gradle.org/). To publish an update to the plugin, add the following lines to the home directory `gradle.properties` file:
```

Comment:
Can you make the path more explicit here so it says "~/.gradle/gradle.properties"?  I was confused when I read this, which is why I had added gradle.properties locally in the project.